users/models.py

```diff
@@ -200,6 +200,9 @@ class User(BaseUser):
 
     badges = models.ManyToManyField(Badge)
     github_username = models.CharField(_("github username"), max_length=100, blank=True)
+    is_commit_author_name_overridden = models.BooleanField(
+        default=False, help_text="Select to override the commit author with Username"
```

Comment:
If you agree with changing the form label for display name, this help text will need to be changed as well.